Recommend using pre-built Docker image (#614)
elastic/eland: README.md

### Docker

> Users wishing to use Eland without installing it, in order to just run the available scripts, can use the Docker
> image.
> It can be used interactively:

Reviewer: I know it's not your sentence originally, but we can improve it now if you don't mind:
```suggestion
If you want to use Eland without installing it just to run the available scripts, use the Docker
image.
It can be used interactively:
```
